### PR TITLE
Run Link Validator as Github action

### PR DESCRIPTION
**Pull request by ennru**
This is an experiment to use my little hobby project to validate links in the Platform Guide.

The report will only show in the log and does not fail for missing links right now.

### CODE REVIEW

File: .github/workflows/link-checker.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run Antora
+        run: make all
+
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
```
Comment (ignasi35) [inline .github/workflows/link-checker.yml:25]: Do you prefer `adopt@1.11.0-9` over the _self-maintained_, _"surprise! surprise!"_ `adopt@1.11`?

File: scripts/validator.conf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  ]
+
+  ignore-missing-local-files-regex = ""
+
+  ignore-prefixes = [
+    "https://github.com/"
+  ]
+
+  non-https-whitelist = [
+    "http://www.reactive-streams.org"
```
Comment (ignasi35) [inline scripts/validator.conf:22]: fix the link to use `https` instead ?

File: scripts/validator.conf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  link-mappings = [
+    {
+      prefix = "https://developer.lightbend.com/docs/akka-platform-guide"
+      replace = ""
+    }
+  ]
+
+  ignore-missing-local-files-regex = ""
+
+  ignore-prefixes = [
```
Comment (ignasi35) [inline scripts/validator.conf:17]: IDK what this does but why ignore links to `github`?

**Review by ignasi35 — COMMENTED** [review]
LGTM, I guess.

File: scripts/validator.conf (hunk 3 above)
Comment (ennru) [reply scripts/validator.conf:17]: Removed it, on other repos too many checks against github.com made them block me very fast.

**Pull request merged**